Sources/Routing/Protocols/ViewLifeCycle.swift

```diff
@@ -0,0 +1,11 @@
+//
+//  Created by Ahmed Fathy on 18/12/2023.
+//
+
+import SwiftUI
+
+public protocol ViewLifeCycle: ObservableObject {
```

Comment:
```suggestion
public protocol Dismissable: ObservableObject {
```

I think this naming is more accurate.